.coveralls.yml

```diff
@@ -0,0 +1 @@
+repo_token: HohKQL7S1Z1OtiHb2oMHEAOqG1iRGsDUq
```

Comment:
I added this to .gitignore but it didn't hide it. I will find another way to hide this. Thanks.